File: Chapter03/Java/TestResources/app/src/main/java/com/ai/android/book/resources/HelloWorldActivity.java
package com.ai.android.book.resources;

import android.app.Activity;
import android.os.Bundle;
import android.widget.TextView;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

public class HelloWorldActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.main);
        TextView tv = findViewById(R.id.text1);
        tv.setText("Try this text instead");
    }
}

File: Chapter03/Kotlin/TestResources/app/src/main/java/com/ai/android/book/resources/HelloWorldActivity.kt
package com.ai.android.book.resources

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.TextView

class HelloWorldActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.main)

        val tv = findViewById<TextView>(R.id.text1)
        tv.text = "Try this text instead"
    }
}